Code/Framework/AzCore/AzCore/std/string/string.h

```diff
@@ -145,11 +145,17 @@ namespace AZStd
             assign(rhs, rhsOffset, count);
         }
 
+        // Constructor #11
```

Comment:
question: Just curious why this constructor gets a special number and the others do not?

Reply:
Solely because this function was just added.

Reply:
Haha okay cool, thanks for checking